game_patch/graphics/d3d11/gr_d3d11_mesh.cpp

```diff
@@ -389,10 +391,28 @@ namespace df::gr::d3d11
         render_context_.set_index_buffer(v3d_ib_.buffer());
 
         auto render_cache = reinterpret_cast<MeshRenderCache*>(lod_mesh->render_cache);
+
+        const bool ir_scanner = (params.flags & MRF_SCANNER_1) != 0;
+        if (!ir_scanner) {
+            rf::Vector3 ambient_light{0.f, 0.f, 0.f};
+            light_get_ambient(&ambient_light.x, &ambient_light.y, &ambient_light.z);
```

Comment:
what's this code? why all the hardcoded stuff? Is it because Goober thinks lighter = better so we just add 40 to ambient light?
I don't want to change game lighting just because Alpine does so. I want it to look as intended by original developers.

Reply:
That is in `render_v3d_vif` which is for e.g. item pickups. Characters are lighted in `render_character_vif`.